app/src/main/java/com/example/sharingang/UserProfileFragment.kt

```diff
@@ -83,13 +85,25 @@ class UserProfileFragment : Fragment() {
             binding.nameText,
             binding.textEmail,
             binding.applyholder,
+            binding.ratingTextview,
+            binding.applyholder,
             binding.btnReport
         )
         for (view: View in fields) {
             view.visibility = View.GONE
         }
     }
 
+    private fun setupRatingView(){
+        userViewModel.refreshRating(shownUserProfileId)
+        userViewModel.rating.observe(viewLifecycleOwner, {
+            var text = resources.getString(R.string.default_rating)
```

Comment:
This variable looks unused

Reply:
oops was meant to be set to ratingTextview